src/electron/office/office_web_plugin.cc

```diff
@@ -291,8 +300,22 @@ blink::WebInputEventResult OfficeWebPlugin::HandleKeyEvent(
     }
   }
 
-  int lok_key_code =
-      office::DOMKeyCodeToLOKKeyCode(event.dom_code, event.GetModifiers());
+  int modifiers = event.GetModifiers();
+
+  if (is_mac) {
+    int result = 0;
+
+    if (modifiers & blink::WebInputEvent::Modifiers::kShiftKey)
+      result |= blink::WebInputEvent::Modifiers::kShiftKey;
+    if (modifiers & blink::WebInputEvent::Modifiers::kAltKey)
+      result |= blink::WebInputEvent::Modifiers::kAltKey;
+    if (modifiers & blink::WebInputEvent::Modifiers::kMetaKey)
+      result |= blink::WebInputEvent::Modifiers::kControlKey;
+
+    modifiers = result;
```

Comment:
```suggestion
    // unset ctrl
    modifiers &= ~blink::WebInputEvent::Modifiers::kControlKey;
    // set ctrl if meta is set
    if (modifiers & blink::WebInputEvent::Modifiers::kMetaKey) {
      modifiers |= blink::WebInputEvent::Modifiers::kControlKey;
      modifiers &= ~blink::WebInputEvent::Modifiers::kMetaKey;
    }
```

Instead of manually copying each bit, just copy the int and mask off the bits you don't want. `&` masks bits, you want everything but a single bit, so bit flip (`~`) the unwanted flag and apply it as a mask.

Reply:
Thanks for that! I was trying to do that earlier but I was using << and >> to do it which obviously wasn't working